src/ImmersedBoundaries/grid_fitted_bottom.jl

```diff
@@ -34,7 +34,19 @@ const GFBIBG = ImmersedBoundaryGrid{<:Any, <:Any, <:Any, <:Any, <:Any, <:GridFit
 """
     GridFittedBottom(bottom_height, [immersed_condition=CenterImmersedCondition()])
 
-Return an immersed boundary with an irregular bottom fit to the underlying grid.
+Return a bottom immersed boundary.
+
+Keyword Arguments
+=================
+
+* `bottom_height`: an array or function that gives the height of the
+                   bottom in absolute ``z`` coordinates.
+
+* `immersed_condition`: determine whether the part of the domain that is 
+                        immersed are all the cell centers that lie below
+                        `bottom_height` (`CenterImmersedCondition()`; default)
+                        or all the cell faces that lie below `bottom_height`
+                        (`InterfaceImmersedCondition()`).
```

Comment:
Probably it should be noted that the only purpose of this is to allow `GridFittedBottom` and `PartialCellBottom` to have the same behavior when the minimum fractional cell height for partial cells is set to 0.

Reply:
Hard to concept to convey simply, but perhaps the main point is that most people don't care about this and shouldn't expend too much energy trying to understand it.

Reply:
okie!